modules/mobile/app/controllers/mobile/v0/decision_letters_controller.rb

```diff
@@ -22,6 +25,17 @@ def download
 
       private
 
+      def log_decision_letters(list)
+        return nil if list.empty?
+
+        Rails.logger.info('MOBILE DECISION LETTERS COUNT',
+                          user_uuid: @current_user.uuid,
+                          user_icn: @current_user.icn,
```

Comment:
Are the `uuid` and `icn` of a user okay to be present in logs?

Reply:
I see some examples elsewhere in vets-api with both values being present. I dont see an issue with it but can remove the uuid if necessary. I mostly had it there as a backup in case I was having trouble matching users by ICN.

Reply:
We've logged this info in the past. Is there somewhere we can confirm or someone we can confirm with?

Reply:
ok, probably fine then. But I'm not sure if there is a master list somewhere. Let me ask :)

Reply:
PII list!
https://dsva.slack.com/archives/C01CJV0L9PS/p1696616692397599

ICN is on there but apparently [that is a recent change](https://dsva.slack.com/archives/C01CJV0L9PS/p1702320744099689?thread_ts=1702320473.950679&cid=C01CJV0L9PS).

Reply:
these values are required and are used for all vets-api applications. Until a policy is declared in which ICN is stated that it cannot be stored within our logs, I recommend we allow this to continue. I can source 60 other apps within vets-api that store this. This isnt to question you @rmtolmach, its to support you in allowing it.

Reply:
_gulp_. I opened a can of worms with this one. Can yall chime in here? @aherzberg @joeniquette  https://dsva.slack.com/archives/C01CJV0L9PS/p1702320473950679